.github/workflows/ci_cron_weekly.yml

```diff
@@ -30,33 +30,35 @@ env:
 jobs:
   tests:
     if: (github.repository == 'astropy/astropy' && (github.event_name == 'schedule' || github.event_name == 'push' || contains(github.event.pull_request.labels.*.name, 'Extra CI')))
-    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
+    runs-on: ubuntu-latest
     env:
       ARCH_ON_CI: "normal"
-    with:
-      submodules: false
-      coverage: codecov
-      libraries: |
-        apt:
-          - language-pack-de
-          - tzdata
-          - graphviz
+    steps:
+      - uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
```

Comment:
I guess this doesn't raise syntax error but also doesn't work. @ConorMacBride , how do I set env var using this workflow template? Thanks!

Reply:
All I want to do is set `ARCH_ON_CI` to `"normal"` for this job.

Reply:
It's not possible. That's one of the limitations of reusable workflows (https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations). There are hacks (e.g. https://github.com/OpenAstronomy/github-actions-workflows/pull/56) but was hoping for a neater solution.

Reply:
@ConorMacBride , thanks for the quick reply! As long as it works, even if hacky, is better than nothing running at all. I can close this PR without merge if you or @Cadair can open something that works to supersede this. Thank you!

Reply:
Actually... maybe I should remove this openastronomy thingy completely from weekly cron job and do it the old way until y'all can figure something out. Then it won't be rushed.

Reply:
https://github.com/astropy/astropy/pull/14077